rules/windows/defense_evasion_microsoft_defender_tampering.toml

```
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/11/22"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when one or more features on Microsoft Defender is disabled.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Microsoft Defender Tampering"
+references = [
+    "https://www.tenforums.com/tutorials/32236-enable-disable-microsoft-defender-pua-protection-windows-10-a.html",
+    "https://www.tenforums.com/tutorials/104025-turn-off-core-isolation-memory-integrity-windows-10-a.html",
+    "https://www.tenforums.com/tutorials/105533-enable-disable-windows-defender-exploit-protection-settings.html",
+    "https://www.tenforums.com/tutorials/123792-turn-off-tamper-protection-microsoft-defender-antivirus.html",
+    "https://www.tenforums.com/tutorials/51514-turn-off-microsoft-defender-periodic-scanning-windows-10-a.html",
+    "https://www.tenforums.com/tutorials/3569-turn-off-real-time-protection-microsoft-defender-antivirus.html",
+    "https://www.tenforums.com/tutorials/99576-how-schedule-scan-microsoft-defender-antivirus-windows-10-a.html",
+]
+risk_score = 47
+rule_id = "fe794edd-487f-4a90-b285-3ee54f2af2d3"
+severity = "medium"
```

```suggestion
risk_score = 21
rule_id = "fe794edd-487f-4a90-b285-3ee54f2af2d3"
severity = "low"
```
I think we need to update the severity due to the noise